### PR TITLE
Update svgr codemod to output TS for tsx inputs

### PR DESCRIPTION
**Pull request by Tobbe**
If an svg is imported in a .tsx file the user probably want the generated SVG component file to be using TypeScript. That's what this PR does.

### CODE REVIEW

File: packages/codemods/src/codemods/v6.x.x/replaceComponentSvgs/replaceComponentSvgs.ts [hunk 1]
```diff
@@ -17,6 +26,7 @@ async function convertSvgToReactComponent(
     {
       jsxRuntime: 'automatic',
       plugins: ['@svgr/plugin-jsx'],
+      typescript,
```
Comment (Tobbe) [inline packages/codemods/src/codemods/v6.x.x/replaceComponentSvgs/replaceComponentSvgs.ts:29]: This, together with the `const isTS =` code below is the main change for this PR.
The rest is mostly changes to make the code easier to read and understand

File: packages/codemods/src/codemods/v6.x.x/replaceComponentSvgs/replaceComponentSvgs.ts [hunk 2]
```diff
@@ -111,15 +122,20 @@ export default async function transform(file: FileInfo, api: API) {
 
         // The absolute path to the new file
         const outputPath = path.join(
-          path.dirname(filePath),
```
Comment (Tobbe) [inline packages/codemods/src/codemods/v6.x.x/replaceComponentSvgs/replaceComponentSvgs.ts:114]: Reading this code in isolation it was hard to know what `filePath` was. Path to the file we're parsing? Path to the svg we found in that file? Path to the file we're going to generate?

This change hopefully makes it more clear that it's the path to the svg file the code is importing

File: packages/codemods/src/codemods/v6.x.x/replaceComponentSvgs/replaceComponentSvgs.ts [hunk 3]
```diff
@@ -25,16 +35,16 @@ async function convertSvgToReactComponent(
...
+  console.log()
   console.log(`SVG converted to React component: ${outputPath}`)
 }
 
 export default async function transform(file: FileInfo, api: API) {
   const j = api.jscodeshift
-
   const root = j(file.source)
 
-  // Do this lazily
```
Comment (Tobbe) [inline packages/codemods/src/codemods/v6.x.x/replaceComponentSvgs/replaceComponentSvgs.ts:36]: The cost of importing `getPaths` has already been paid by `replaceComponentSvgs.yargs.ts`, so moved this to a regular import, so you don't have to wonder why we're doing a lazy import.

File: packages/codemods/src/codemods/v6.x.x/replaceComponentSvgs/replaceComponentSvgs.ts [hunk 4]
```diff
@@ -25,16 +35,16 @@ async function convertSvgToReactComponent(
 
   await fs.writeFile(outputPath, jsCode)
 
+  console.log()
```
Comment (Tobbe) [inline packages/codemods/src/codemods/v6.x.x/replaceComponentSvgs/replaceComponentSvgs.ts:38]: Extra blank line needed for Listr to not mess up the output

**Review by jtoar — APPROVED** [review]
Worked great for me, thanks @Tobbe!

**Pull request merged**